timemachine/cpp/CMakeLists.txt

```diff
@@ -12,14 +12,7 @@ find_package(PythonLibs 3.7 REQUIRED)
 string(APPEND CMAKE_CXX_FLAGS "-Werror -Wall -Wextra")
 
 string(APPEND CMAKE_CUDA_FLAGS "-O3 -lineinfo")
-string(APPEND CMAKE_CUDA_FLAGS
-  " -Werror all-warnings"
-  " -Wall"
-  " -Wno-sign-compare"
-  " -Wextra"
-  " -Wno-unused-parameter"
-  " -Wreorder"
-)
+
```

Comment:
Wasn't mentioned in PR description -- confirm if this is part of the current PR?

Reply:
good catch, this was a bad merge conflict resolution. will fix